### DISCUSSION TITLE
Unable to create disabled panos_nat_rule

### DISCUSSION THREAD

**Discussion by Nothing4You** [PaloAltoNetworks/pan-os-ansible#209] [Ideas]
### Is your feature request related to a problem?

I'm unable to create disabled nat rules in a single task.
Using 2 tasks results in always having 2 changed tasks.

### Describe the solution you'd like

Either consider states `enable` and `disable` similar to `present` and create rules accordingly or introduce a separate parameter like `disabled` in `panos_security_rule`.

### Describe alternatives you've considered

Using 2 tasks, first one with `state: "present"`, second task with `state: "disabled"`.
This results in both tasks being changed in each run.

### Additional context

I need disabled rules to ensure being able to set up the correct order using `location` and `existing_rule` even if some rules in the list are unused at the time.

```
TASK [paloalto : nat rule foo]
fatal: [pa]: FAILED! => {"changed": false, "msg": "Rule \"foo\" not present"}
```

**Comment by welcome-to-palo-alto-networks** [comment]
:tada: Thanks for opening your first issue here! Welcome to the community!
